src/components/Book/Book.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function Book() {
+  return (
+    <div className="book">
+      <header className="book-header">
+        <h2 className="book-title">Hello World!</h2>
+      </header>
+    </div>
+  );
+}
```

Comment:
- [ ] Please make sure that this Book component takes book data (Create book data if you don't have it) as a prop and renders the title and the author. Also, ensure that this component renders the book title and author with a button for removing a book. :blush:

Reply:
Done! Thanks